src/components/views/elements/crypto/VerificationQRCode.js

```diff
@@ -31,13 +34,81 @@ export default class VerificationQRCode extends React.PureComponent {
         // User verification use case only
         secret: PropTypes.string,
         otherUserKey: PropTypes.string, // Base64 key being verified
-        requestEventId: PropTypes.string,
+        otherUserDeviceKey: PropTypes.string, // Base64 key of the other user's device (optional)
```

Comment:
might be helpful to clarify here that this is what we *think* their device key is (presumably?)